cub/cub/device/device_copy.cuh

```diff
@@ -42,8 +42,7 @@ _CCCL_IMPLICIT_SYSTEM_HEADER
 
 #include <cub/device/dispatch/dispatch_batch_memcpy.cuh>
 
-#include <thrust/system/cuda/detail/core/triple_chevron_launch.h>
-
+#include <cub/detail/triple_chevron_launch.cuh>
```

Comment:
suggestion: I don't think this include is needed by this header. Please, remove it.